src/test/javascript/spec/helpers/mocks/service/mock-clipboard-item.ts

```diff
@@ -1 +1,7 @@
-export class MockClipboardItem {}
+export class MockClipboardItem {
+    types: string[];
+
+    getType(_type: string): Promise<Blob> {
```

Comment:
I prefix this with _ here to avoid a warning for an unused variable while still preserving the signature

Reply:
I think this might be worth adding a comment in the code here as well 😅